upgrade.go

```diff
@@ -53,10 +55,11 @@ func (b *versionedKVBackend) Upgrade(ctx context.Context, s logical.Storage) err
 		return errors.New("upgrade already in process")
 	}
 
-	// If we are a replication secondary, wait until the primary has finished
+	// If we are a replication secondary or performance standby, wait until the primary has finished
 	// upgrading.
-	if !b.System().LocalMount() && b.System().ReplicationState().HasState(consts.ReplicationPerformanceSecondary|consts.ReplicationPerformanceStandby) {
-		b.Logger().Info("upgrade not running on performace replication secondary")
+	if (!b.System().LocalMount() && replState.HasState(consts.ReplicationPerformanceSecondary)) ||
+		replState.HasState(consts.ReplicationPerformanceStandby) {
+		b.Logger().Info("upgrade not running on performance replication secondary or performance standby")
```

Comment:
Does this say it's not going to run and then, you know, run?

Reply:
It doesn't run the upgrade, but it does wait for an upgrade being done by another node to complete.

Reply:
Oh, I need to stop looking at diffs on phones :-)